### DISCUSSION TITLE
After update expo got error Error: Invalid hook call

### DISCUSSION THREAD

**Discussion by rickyanwar** [lawnstarter/react-native-picker-select#518] [Q&A]
i have update myexpo project
old expo

not any issue, now i have issue  
 ERROR  Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.

package.json
  "react-native": "0.72.4",
    "expo": "^49.0.0",
        "react-native-picker-select": "^8.0.4",

**Comment by Yormungand** [comment]
https://github.com/lawnstarter/react-native-picker-select/issues/486#issuecomment-1475912811

I'm facing same issue and I tried above method but didn't work.
